### PR TITLE
Support EAP-PEAP and EAP-TLS for CLI and VPN authentication

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Support EAP-PEAP MSCHAPv2 and EAP-TLS for CLI and VPN authentication

# Impacts
CLI and VPN authentication

# Delete branch after merge
YES

# NEWS file entries
## Enhancements
* Added support for EAP-PEAP MSCHAPv2 and EAP-TLS for CLI and VPN RADIUS authentication

### CODE REVIEW

**Comment by lyubomirtraykov** [comment]
Please add support for EAP-MSCHAPv2 (EAP-Type = 29), because it is the only supported method for Android StrongSwan client.

**Comment by julsemaan** [comment]
> Please add support for EAP-MSCHAPv2 (EAP-Type = 29), because it is the only supported method for Android StrongSwan client.

I can only add what I can test but I think this is handled by the MSCHAP-Challenge test

File: lib/pf/radius.pm [hunk 1]
```diff
@@ -169,7 +170,6 @@ sub authorize {
         ifIndex => $port,
         ifDesc => $ifDesc,
         user_name => $user_name,
-        username => $user_name,
```
Comment (fdurand) [inline lib/pf/radius.pm]: We have to keep it since we use it in the radius filter.

Comment (julsemaan) [reply lib/pf/radius.pm]: my bad, this shouldn't have been part of the PR

**Comment by julsemaan** [comment]
Adjusted

**Pull request merged**